drivers/scheduler/dcos/dcos.go

```diff
@@ -394,6 +394,16 @@ func (d *dcos) Describe(ctx *scheduler.Context) (string, error) {
 	return "", nil
 }
 
+func (d *dcos) ScaleApplication(ctx *scheduler.Context, deploymentScaleFactorMap map[string]int32) error {
+	//TODO implement this method
+	return nil
+}
+
+func (d *dcos) GetNewScaleFactorMap(ctx *scheduler.Context, factor int32) (map[string]int32, error) {
```

Comment:
once you change this to return the current scale factor.. you can call it GetScaleFactorMap instead of New

Reply:
Fixed.